### ISSUE TITLE
Critical Time Step Calculation - Going Wrong

### ISSUE THREAD

**Issue by uppatispr** [peridigm/peridigm#14]
Hi,

I'm trying to use a python generated text file input mesh to run a simulation in Peridigm.

I'm running into the problem of incorrect stable time step calculation. 

-- Peridigm
-- version 1.4.1

Time step (seconds):
  Stable time step    1e+50
  User time step      not provided
  Safety factor       0.7
  Time step           7e+49

Total number of time steps 0

My input mesh (livermore_cylinder.txt), nodesets and input deck (livermore.txt, converted from livermore.peridigm for attaching here on GitHub) are attached. I had a cursory look at the Peridigm_CriticalTimeStep.cpp file, looks like Peridigm is using the default 1e50 value. What might cause this? Note: I'm using SI system of units here.

On the other hand, when I convert the mesh generator and input deck values into the cgs system of units, the proper calculation was done:

-- Peridigm
-- version 1.4.1

Time step (seconds):
  Stable time step    4.66567e-07
  User time step      not provided
  Safety factor       0.75
  Time step           3.49926e-07

Total number of time steps 28577

Seems very strange to me. 

[livermore_cylinder.txt](https://github.com/peridigm/peridigm/files/361881/livermore_cylinder.txt)
[nodeset_full.txt](https://github.com/peridigm/peridigm/files/361879/nodeset_full.txt)
[nodeset1.txt](https://github.com/peridigm/peridigm/files/361880/nodeset1.txt)
[livermore.txt](https://github.com/peridigm/peridigm/files/361884/livermore.txt)

**Comment by djlittl** [comment]
Hopefully you saw the earlier reply to this question.  The volumes are all zero in livermore_cylinder.txt.  This is likely the cause of the problem.  Perhaps your script is truncating the numerical values such that they are 0.00000000 when you use one set of units, but the values are being output as nonzero when you use different units.  Whatever the root cause, you cannot have a node with zero volume.

Dave

**Comment by uppatispr** [comment]
Hi Dr. Littlewood,

I did see your email in response to my query to the Peridigm developers mailing list yesterday. It just so happened that I had just submitted it as an issue on here, when I saw your response this morning. So, I realized my mistake and tried to delete the issue thread I started. However, I couldn't find any delete option, so I just closed the issue instead. If you are able to, go ahead and delete it on your end.

Thank you for your help in identifying the error as well. I really appreciate it. 

Sai
